README.md

```diff
@@ -41,17 +41,13 @@ First install the required dependencies through `bundle install`.
 
 The project is configured through environment variables.
 Copy the `.env.sample` configuration file to `.env`.
```

Comment:
this is more of a future suggestion, but it would be cool to have a setup script that does this for people so setup is more painless. Example of a setup script here: https://github.com/18F/dolores-landingham-bot/pull/3